src/assets/scss/application.scss

```diff
@@ -197,11 +197,81 @@ $govuk-new-link-styles: true;
   }
 }
 
-$nav-link-padding: govuk-spacing(3);
+$nav-link-padding: govuk-spacing(4);
 $nav-link-outline-thickness: 2px;
 $nav-active-border-thickness: 5px;
 
-.account-navigation__link {
+/* Combined styles for buttons */
+.account-header-navigation__button,
+.account-navigation__button {
+  border: 0 none;
+  cursor: pointer;
+  background-color: unset;
```

Comment:
My suggestion would be to change this to `background-color: transparent` which should produce the same outcome, but has better cross-browser support, so the "Sign out" link would be readable when viewed on a browser that doesn't support `unset`, whereas right now it's not:
![unsetie11](https://github.com/govuk-one-login/di-account-management-frontend/assets/7116819/d0ca0239-9e96-403c-98af-5d505c0d3cc9)